src/materials/point-cloud-material.ts

```diff
@@ -123,14 +129,22 @@ const COLOR_DEFS = {
 const CLIP_MODE_DEFS = {
   [ClipMode.DISABLED]: 'clip_disabled',
   [ClipMode.CLIP_OUTSIDE]: 'clip_outside',
+  [ClipMode.CLIP_INSIDE]: 'clip_inside',
   [ClipMode.HIGHLIGHT_INSIDE]: 'clip_highlight_inside',
+  [ClipMode.CLIP_OUTSIDE_PICK]: 'clip_outside_pick',
 };
 
 export class PointCloudMaterial extends RawShaderMaterial {
   lights = false;
   fog = false;
   numClipBoxes: number = 0;
   clipBoxes: IClipBox[] = [];
+  numClipSpheres: number = 0;
```

Comment:
We don't really need to keep the number of clipping geometries because we can get the `.length` of the arrays when we are setting the uniforms.